cmd/bee/cmd/start.go

```diff
@@ -33,6 +33,131 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type signerConfig struct {
+	keystore         keystore.Service
+	signer           crypto.Signer
+	address          swarm.Address
+	publicKey        *ecdsa.PublicKey
+	libp2pPrivateKey *ecdsa.PrivateKey
+	password         string
+}
+
+func (c *command) configureSigner(cmd *cobra.Command, logger logging.Logger) (config *signerConfig, err error) {
```

Comment:
I would put this to the end of this file or even in init.go file, but it is a very minor thing, nothing crucial, just a suggestion.

Reply:
I just put it at the end of start.go